router/utils.gno

```diff
@@ -92,6 +94,20 @@ func min(a, b int) int {
 	return b
 }
 
+func i256Min(x, y *i256.Int) *i256.Int {
+	if x.Lt(y) {
+		return x
+	}
+	return y
+}
+
+func i256Max(x, y *i256.Int) *i256.Int {
+	if x.Gt(y) {
+		return x
+	}
+	return y
+}
+
```

Comment:
needs to be moved to the `common` package. maybe `common/math`?

Reply:
How about we make `gnsmath `and `gnsutils `in the root folder and put math related files to `gnsmath`, util related files to `gnsutils`?

Reply:
related with https://github.com/gnoswap-labs/gnoswap/pull/444